docs/advanced_features.md

```diff
@@ -0,0 +1,71 @@
+# Advanced features
+
+## SPARQL+Text and SPARQL Autocompletion
+
+On top of the vanilla SPARQL functionality, QLever allows so-called SPARQL+Text
+queries on a text corpus linked to a knowledge base via entity recognition.  For
+example, the following query find all mentions of astronauts next to the words
+"moon" and "walk*" in the text corpus:
+
+    SELECT ?a TEXT(?t) SCORE(?t) WHERE {
+        ?a <is-a> <Astronaut> .
+        ?t ql:contains-entity ?a .
+        ?t ql:contains-word "walk* moon"
+    } ORDER BY DESC(SCORE(?t))
+
+Such queries can be simulated in standard SPARQL, but only with poor
+performance, see the CIKM'17 paper above.  Details about the required input data
+and the SPARQL+text query syntax and semantics can be found
+[here](docs/sparql_plus_text.md).
+
+QLever also supports efficient SPARQL autocompletion.  For example, the
+following query yields a list of all predicates associated with persons in the
+knowledge base, ordered by the number of persons which have that predicate.
+
+    SELECT ?predicate (COUNT(?predicate) as ?count) WHERE {
+      ?x <is-a> <Person> .
+      ?x ql:has-predicate ?predicate
+    }
+    GROUP BY ?predicate
+    ORDER BY DESC(?count)
+
+Note that this query could also be processed by standard SPARQL simply by
+replacing the second triple by ?x ?predicate ?object. However, that query is
+bound to produce a very large intermediate result (all triples of all persons)
+with a correspondingly huge query time.  In contrast, the query above takes only
+~ 100ms on a standard Linux machine for a dataset with ~ 360 million triples and
+~ 530 million text records.  More details on this feature set will be provided
+here soon.
+
+## Statistics:
+
+You can get stats for the currently active index in the following way:
+
+    <server>:<port>/?cmd=stats
+
+This query will yield a JSON response that features:
+
+* The name of the KB index
+* The number of triples in the KB index
+* The number of index permutations build (usually 2 or 6)
+* The numbers of distinct subjects, predicates and objects (only available if 6 permutations are built)
+* The name of the text index (if one is present)
+* The number of text records in the text index (if text index present)
+* The number of word occurrences/postings in the text index (if text index present)
+* The number of entity occurrences/postings in the text index (if text index present)
+
+
+The names of a index is the name of the input nt file (and wordsfile for the text index) but can also be specified manually while building an index.
+Therefore, IndexbuilderMain takes two optional arguments: --text-index-name (-T) and --kb-index-name (-K).
+
+
+## Send vs Compute
+
+Currently, QLever does not compute partial results if there is a LIMIT modifier.
+However, strings (for entities and text excerpts) are only resolved for those items that are actually send.
+Furthermore, in a UI, it is usually beneficial to get less than all result rows by default.
```

Comment:
I would restructure this sentence to avoid the `less than all`, as it sounds akward to me. For example:
`Furthermore, a UI usually only [requires/wants] a limited amount of rows at a time.`